railties/test/application/configuration_test.rb

```diff
@@ -1890,6 +1890,38 @@ def index
       assert_equal "https://example.org/", last_response.location
     end
 
+    test "ActiveSupport::MessageEncryptor.use_authenticated_message_encryption is true by default for new apps" do
+      app "development"
+
+      assert_equal true, ActiveSupport::MessageEncryptor.use_authenticated_message_encryption
+    end
+
+    test "ActiveSupport::MessageEncryptor.use_authenticated_message_encryption is false by default for upgraded apps" do
+      remove_from_config '.*config\.load_defaults.*\n'
+      add_to_top_of_config <<-RUBY
+        config.load_defaults 5.1
```

Comment:
Why did you use `load_defaults` here? Seems 5.1 default is unrelated.

Reply:
In real apps, we only expect this option to be set after an upgrade from 5.1. I wrote these three test cases to cover all of the expected states: a new Rails 5.2 app, an upgraded Rails 5.1 app before enabling the new default, and an upgraded Rails 5.1 app after enabling it.

However, you're right that all we need to demonstrate the problem is a single test that sets the option in an initializer. Would you prefer that, even if it doesn't represent how we expect the option to be used in real apps?

Reply:
Oh, I realised what you actually meant. Ignore what I wrote above ☝️ 🤦‍♂️ 

I thought it would be more accurate to simulate an app generated on 5.1 upgrading to 5.2, but an app generated on 5.0 would also have this problem, and wouldn't have `load_defaults`.

I've removed the `load_defaults` calls from the tests added here, and another one in the same file that I copied the pattern from. Thanks!